### PR TITLE
chore(install): adjust output to include recommendations

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: internal/install/execution/terminal_reporter.go [hunk 1]
```diff
@@ -65,20 +65,31 @@ func (r TerminalStatusReporter) ReportRecipeAvailable(status *StatusRollup, reci
...
-	msg := `
-  Success! Your data is available in New Relic.
+	recs := status.recommendations()
+
+	if len(recs) > 0 {
+		fmt.Println("  ---")
+		fmt.Println("  Instrumentation recommendations")
+		fmt.Println("  We discovered some additional instrumentation opportunities")
+
+		for _, recommendataion := range recs {
```
Comment (ctrombley) [inline internal/install/execution/terminal_reporter.go:79]: nit: s/recommendataion/recommendation/

File: internal/install/execution/terminal_reporter.go [hunk 2]
```diff
@@ -65,20 +65,31 @@ func (r TerminalStatusReporter) ReportRecipeAvailable(status *StatusRollup, reci
...
 	}
 
-	msg := `
-  Success! Your data is available in New Relic.
+	recs := status.recommendations()
+
+	if len(recs) > 0 {
+		fmt.Println("  ---")
+		fmt.Println("  Instrumentation recommendations")
+		fmt.Println("  We discovered some additional instrumentation opportunities")
```
Comment (ctrombley) [inline internal/install/execution/terminal_reporter.go:77]: nit: Maybe add a colon at the end of the message?

File: internal/install/execution/terminal_reporter.go [hunk 3]
```diff
@@ -65,20 +65,31 @@ func (r TerminalStatusReporter) ReportRecipeAvailable(status *StatusRollup, reci
...
 
 func (r TerminalStatusReporter) ReportComplete(status *StatusRollup) error {
-
 	if status.hasFailed() {
 		return fmt.Errorf("one or more integrations failed to install, check the install log for more details: %s", status.LogFilePath)
 	}
 
-	msg := `
-  Success! Your data is available in New Relic.
+	recs := status.recommendations()
```
Comment (ctrombley) [inline internal/install/execution/terminal_reporter.go:72]: Should this include skipped as well as recommended (APM) recipes?

File: internal/install/execution/terminal_reporter.go (hunk 1 above)
Comment (zlesnr) [reply internal/install/execution/terminal_reporter.go]: 👍🏼

File: internal/install/execution/terminal_reporter.go (hunk 2 above)
Comment (zlesnr) [reply internal/install/execution/terminal_reporter.go]: 👍🏼

File: internal/install/execution/terminal_reporter.go (hunk 3 above)
Comment (zlesnr) [reply internal/install/execution/terminal_reporter.go:72]: Maybe not just yet.

**Review by sanderblue — APPROVED** [review]

**Pull request merged**